doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -132,7 +158,7 @@ Optional steps for setting up the S3 Docker DCM Variant
 
     - S3 bucket for Dataverse
 
-      - ``/usr/local/glassfish4/glassfish/bin/asadmin create-jvm-options "-Ddataverse.files.s3-bucket-name=iqsstestdcmbucket"``
+      - ``/usr/local/glassfish4/glassfish/bin/asadmin create-jvm-options "-Ddataverse.files.s3.bucket-name=iqsstestdcmbucket"``
```

Comment:
Why was this changed from a hyphen to a period?

Reply:
When there was only one possible s3-bucket-name, a hyphen made sense. Now that there can be multiple S3 stores, we need options for each store's bucket name. I chose to standardize on
-Ddataverse.files.<id>.bucket-name 
and to have all of the options for all three types of stores be of the form
-Ddataverse.files.<id>.<option name>
rather than have a mix of . and hyphens depending on type. 
If there's an argument for a better convention, I'm open.